rfcs/rfc0018.md

```diff
@@ -0,0 +1,69 @@
+# A feature to automatically "yield true" at the end of a file
+
+## Preamble
+
+    Author:  Curtis "Ovid" Poe <curtis.poe@gmail.com>
+    Sponsor:
+    ID:      0018
+    Status:  Draft
+
+## Abstract
+
+This RFC proposes a `yield_true` feature. When used, the current Perl _file_
+containing `use feature 'yield_true'l` will automatically return a true value
+after successful compilation, eliminating the neeed for a "1" (or other true
+value) at the end of the file.
+
+## Motivation
+
+Eliminate the need for a true value at the end of a Perl file.
+
+## Rationale
+
+There's no need to have a true value be hard-coded in our files that we
+`use`. Further, newer programmers can get confused because sometimes code
+_doesn't_ end with a true value but nonetheless compiles just fine because
+_something_ in the code returned a true value and the code compiles as a
+side-effect.
+
+
+## Specification
+
+    use feature 'yield_true';
+
+Code using the above does not need to return a magic true value when compiled.
+
+## Backwards Compatibility
+
+There are no compatibility concerns I'm aware of.
+
+## Security Implications
+
+None expected.
+
+## Examples
+
+None. See above.
+
+## Prototype Implementation
+
+None.
+
+## Future Scope
+
+I believe this is complete.
```

Comment:
The intention of having it be a named feature, is that a future version bundle will include it, so you'll get it automatically with, perhaps,

```perl
use v5.40;
```